org.gnomeradio.Radio.json

```diff
@@ -0,0 +1,55 @@
+{
+  "app-id" : "org.gnomeradio.Radio",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "44",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "gnome-radio",
+  "rename-icon" : "gnome-radio",
+  "copy-icon" : true,
+  "rename-desktop-file" : "gnome-radio.desktop",
+  "rename-appdata-file" : "gnome-radio.appdata.xml",
+    "finish-args" : [
+     "--share=network",
+     "--socket=wayland",
+     "--socket=fallback-x11",
+     "--socket=pulseaudio",	
+     "--socket=session-bus",
```

Comment:
one more time. This is not allowed.

Reply:
Works for me.  Don't change something that runs with Flatpak.